### PR TITLE
Remove adrenal implant and associated reagent

### PR DESCRIPTION
**Pull request by kriskog**
## About The Pull Request
People are still regularly whinging about the adrenal implant rebalance I did a year and a half ago, and it is high time I correct my mistakes. This PR fully removes the adrenal implant and its associated chemical. Since nanite adrenals rely on the same chemical, those are gone too.

## Why It's Good For The Game
Old adrenals were not in a good state, and ended up being a "must buy" to succeed as a traitor. New adrenals are more acceptable from a balance standpoint but requires a bit of timing and at least two braincells to use effectively. In other words, new adrenals are completely unwanted by our playerbase. I do not believe the existence of adrenals is worth the balance implications of "fixing" them.

## Changelog
:cl: Skoglol
del: Removes the adrenal implant and the adrenal nanite program.
/:cl:

### CODE REVIEW

**Comment by Pugie** [comment]
haha

**Comment by Fhaxaris** [comment]
> Fixes all adrenal implant issues

just fucking removes them holy based

**Comment by MTandi** [comment]
I've been using the Adrenaline Burst Nanite program that works automatically when you're attacked, to have a chance to run away from traitor during ambush. But people on Terry were only angry because it made them drop weapons and unable to click on target as they fight back rather than run away.

I figured that it'd be better to let them enable it with dermal button, but I rarely seen anyone using it.

Kinda sad though, I liked the design of it for the use case when you're running away from attacker.

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 1]
```diff
@@ -1320,38 +1320,6 @@
...
-			to_chat(M, "<span class='notice'>Your hands spaz out and you drop what you were holding!</span>")
-	M.adjustStaminaLoss(-10, 0)
-	M.Jitter(10)
-	M.Dizzy(15)
-
-/datum/reagent/medicine/badstims/on_mob_metabolize(mob/living/L)
-	..()
-	ADD_TRAIT(L, TRAIT_SLEEPIMMUNE, type)
-	ADD_TRAIT(L, TRAIT_STUNRESISTANCE, type)
-	L.add_movespeed_modifier(/datum/movespeed_modifier/reagent/badstims)
```
Comment (Fikou) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm:1343]: i think you forgot to remove that movespeed modifier

Comment (kriskog) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm:1343]: you mean line 1350?

**Comment by Pugie** [comment]
As chemist I spend about 20-25 minutes making a really autistic pill mix that has essentially the same effects of the adrenal implant but without the item drop and it's really good, other people have said that it's really good and I've blocked stuns pretty well with it. The only major difference is it has higher stam regen (20 stam a tick) and it helps against actual hardstuns but these kind of improvements should be expected when working on something for 20 minutes.
Overall the effects are pretty similar though, so I'm pretty sure on adrenal implant being totally viable. There's tons more unused stuff for traitors that should be removed before this like camera bugs and ai detectors & syndie kit b special those 3 are useless compared to adrenals

**Comment by Tlaltecuhtli** [comment]
lol

**Comment by kriskog** [comment]
> As chemist I spend about 20-25 minutes making a really autistic pill mix that has essentially the same effects of the adrenal implant but without the item drop and it's really good, other people have said that it's really good and I've blocked stuns pretty well with it. The only major difference is it has higher stam regen (20 stam a tick) and it helps against actual hardstuns but these kind of improvements should be expected when working on something for 20 minutes.

This was completely intentional, and was discussed at length in the nerf PR. Something requiring time and effort in chemistry should be better than an instant 8tc buy. I too agree that adrenals in their current iteration are viable, but even people who have had the effects and ways of efficient use described to them fail to make them work. Better we remove them, right now they do not serve the intended purpose and might instead be more of a noobtrap.

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm (hunk 1 above)
Comment (kriskog) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm:1343]: You're right, thanks

**Comment by Pugie** [comment]
That's understandable

**Comment by necromanceranne** [comment]
> Overall the effects are pretty similar though, so I'm pretty sure on adrenal implant being totally viable.

Alternatively the adrenals can finally perish like they should have done the first time. The primary problem with old adrenals has and always been that they dictate a lot of the game balance for their existence by justifying harsher tools to deal with them. When that was happening, they became a mandatory purchase that screws anyone who either doesn't have them or makes any opponent that didn't get them as an option entirely shit out of luck. They were a blight. 

It's probably why it was the last on the list after batons got reworked and tasers were long dead. When adrenals had no relevance, there was no reason for for them to continue to exist as they did. (And hilariously, they were quite a bit weaker in a stamina based meta since they were entirely based around anti-stun over anti-stamina short of blowing excess charges to reverse baton hits, which was dreadfully ineffective)

**Comment by ATH1909** [comment]
this is one of the most passive-aggressive PRs I've ever seen

**Comment by kriskog** [comment]
Sorry, I didnt intend to be passive at all

**Comment by ATH1909** [comment]
this is a big balance implication for nanites, btw- their pseudo-adrenals are pretty much their only answer to/way to heal stamina damage

**Comment by D4C-420** [comment]
maybe remove or limit the chance to drop an item? I mean Ephedrine is not hard to make and has a lower drop chance and it's speed boost is about the same. takes about 7-10 minutes to make a instant ephedrine button and a ton of pills for some chemist to keep up the speed boost. If you don't want Experimental Stimulants to suck take away the shitty drop chance. People are still going to use em for the speed boost.

**Comment by ATH1909** [comment]
> maybe remove or limit the chance to drop an item? I mean Ephedrine is not hard to make and has a lower drop chance and it's speed boost is about the same. takes about 7-10 minutes to make a instant ephedrine button and a ton of pills for some chemist to keep up the speed boost. If you don't want Experimental Stimulants to suck take away the shitty drop chance. People are still going to use em for the speed boost.

you should probably keep the drop chance for nanite adrenals, though, just so that they don't significantly outclass the traitor version

**Comment by kriskog** [comment]
> If you don't want Experimental Stimulants to suck take away the shitty drop chance.

I do want them to suck, relative to the effort alternatives. I just wish our players didnt. :) :) :) :) :) :)

**Comment by D4C-420** [comment]
> you should probably keep the drop chance for nanite adrenals, though, just so that they don't significantly outclass the traitor version
fair enough but if we make them diffrent might was well keep the nanites on experimental simulants and change the Adeline implant to give a smaller amount of regular stimulant's

**Comment by ATH1909** [comment]
> Sorry, I didnt intend to be passive at all



> > If you don't want Experimental Stimulants to suck take away the shitty drop chance.
> 
> I do want them to suck, relative to the effort alternatives. I just wish our players didnt. 🙂🙂🙂🙂🙂🙂

okay, can you tell us who specifically pissed you off? there's clearly someone/something specific that you're reacting to/venting about here.

**Comment by kriskog** [comment]
First line, buddy

> People are still regularly whinging about the adrenal implant rebalance I did a year and a half ago

**Comment by D4C-420** [comment]
> People are still regularly whinging about the adrenal implant rebalance I did a year and a half ago

And that justify removing them because? What are they bitching about?

**Comment by Sparkezel** [comment]
In good old times of permastuns, adrennals used to just make you ignore stuns and stamina damage, now they just make you regen stamina at slower rate and it doesn't pick you up, they also speed you up but you have 30% chance to drop shit in your active weapon - deadly

**Comment by kriskog** [comment]
> 
> 
> > People are still regularly whinging about the adrenal implant rebalance I did a year and a half ago
> 
> And that justify removing them because? What are they bitching about?

I was bullied ok
image

**Comment by D4C-420** [comment]
> People are still regularly whinging about the adrenal implant rebalance I did a year and a half ago

And that justify removing them because? What are they bitching about?

I was bullied ok
image

Understandable obey the birb. But really if you want players to use them just remove the drop chance that's all I see that needs to be removed to get people to make use of them.

**Comment by ATH1909** [comment]
> First line, buddy
> 
> > People are still regularly whinging about the adrenal implant rebalance I did a year and a half ago

ok but who is "people"

**Comment by cacogen** [comment]
>New adrenals are more acceptable from a balance standpoint but requires a bit of timing and at least two braincells to use effectively. In other words, new adrenals are completely unwanted by our playerbase.

How do we heal the division between the server and project? It's absolute.

**Comment by moocowswag** [comment]
You'd think this pr would be alot less split / controvercial / fought if the general community didnt like the implant or stimulants.

Its an "oh no" button that cant be taken from you and can help you run away and avoid getting arrested, its a niche thats extremely valuable on builds/strats that involve indirect combat.

Even if the majority of the playerbase didnt like using it youre destroying it for the minority of players that use it. 

There is a long list of traitor items and many of them are niche / considered bad by the majority and thats the point of the long list so everyone can pick their playstyle so why remove this?

**Comment by HumanNugget** [comment]
Oh no i got killed by adrenal dual saber tator WORDLESSLY
Proceeds to merge request adrenal nerf
Merged
Oh no people dont like my nerf
**People are still regularly whinging about the adrenal implant rebalance**
Feature removed 
Points received
Playerbase trolled
peak maintainer

**Comment by Jared-Fogle** [comment]
Okay

**Comment by AffectedArc07** [comment]
here before the rshoe tide arrives

**Pull request merged**